src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10844,7 +10811,7 @@ void getVRFTUZoneLoad(
 bool getVRFTUNodeNumber(EnergyPlusData &state, int const nodeNumber)
 {
     for (int vrfTUIndex = 1; vrfTUIndex <= state.dataHVACVarRefFlow->NumVRFTU; ++vrfTUIndex) {
-        auto &vrfTU = state.dataHVACVarRefFlow->VRFTU(vrfTUIndex);
+        auto const &vrfTU = state.dataHVACVarRefFlow->VRFTU(vrfTUIndex);
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10847]:(style),[constVariable],Variable 'vrfTU' can be declared as reference to const